README.md

```diff
@@ -77,9 +77,12 @@ Add a Renovate configuration to your project and use
 `"extends": ["github>ridedott/renovate-config"]` to include the options defined
 as defaults in the **./default.json** configuration:
 
+If you are working in a TypeScript project we recommend using the `typescript`
+preset by extending from `gihub>ridedott/renovate-config:typescript` instead.
```

Comment:
```suggestion
preset by extending from `github>ridedott/renovate-config:typescript` instead.
```

Reply:
Had the commit locally but never pushed it 🤦🏽, thanks.